src/components/Property/Property.js

```diff
@@ -41,11 +41,30 @@ export default class Property extends Component {
         <td className="property-info">
           <span>{type}</span>{subtype && <span> of {subtype}</span>}
           {required && <span className="property-required">Required</span>}
+          {enumValues && this.renderEnumValues(enumValues)}
           {description && <div className="property-description">{description}</div>}
         </td>
       </tr>
     );
   }
+
+  /**
+   * Render enum values
+   *
+   * @param {Array} values
+   */
+  renderEnumValues(values) {
+    return (
+      <div>
+        <span>Valid values:</span>
+        {values.map(value => {
+          return (
+            <span className="enum">{value}</span>
```

Comment:
Add a key attribute here to prevent warning Each child in an array or iterator should have a unique "key" prop.